.env.example

```diff
@@ -108,6 +108,11 @@ DROPBOX_OAUTH_SECRET=
 WUNDERLIST_OAUTH_KEY=
 WUNDERLIST_OAUTH_SECRET=
 
+EVERNOTE_OAUTH_KEY=
+EVERNOTE_OAUTH_SECRET=
+# Set to true in development, false in production
```

Comment:
Do you think this should be inferred from Rails.env, or will users sometimes want to run the sandbox in production?